src/executables/master_heartbeat.c

```diff
@@ -2658,12 +2658,14 @@ hb_cluster_load_group_and_node_list (char *ha_node_list, char *ha_replica_list)
 {
   int priority, num_nodes;
   char tmp_string[LINE_MAX];
+  char err_string[LINE_MAX];
   char *p, *savep;
   HB_NODE_ENTRY *node;
 
-  if (ha_node_list == NULL)
+  if (ha_node_list == NULL || ha_node_list[0] == '\0')
```

Comment:
I think we also need to set same ERROR message at the caller (`hb_cluster_initialize ()`)

Reply:
The error message is written to master.err . 
Does caller need to write the same error message to master.err, if the function return error ?

Reply:
In `hb_cluster_initialize()`, the same check as above is performed. If the check fails there, this function will not be called. Therefore, I thought the caller should also output the same error message as above.

Reply:
OK, I got it.
I will modify it. thank your review.